static/usage/v7/content/theming/safe-area/vue.md

```diff
@@ -0,0 +1,38 @@
+```html
+<template>
+  <ion-content>
+    <span>Here's a small text description for the content. Nothing more, nothing less.</span>
+  </ion-content>
+</template>
+
+<script lang="ts">
+  import { IonContent } from '@ionic/vue';
+  import { defineComponent } from 'vue';
+
+  export default defineComponent({
+    components: {
+      IonContent,
+    },
+  });
+</script>
+
+<style scoped>
```

Comment:
vue styles need to not have `scoped` for `:root` to work

Reply:
oh and also a change is required for angular, like what i've done [in my safe area PR](https://github.com/ionic-team/ionic-docs/pull/3105/commits/c41c634e712aee8cfda83faaf2af4a5e7cdfd9bf)